.github/workflows/pr-linter.yml

```diff
@@ -25,8 +68,8 @@ jobs:
       statuses: read
       issues: read
     runs-on: ubuntu-latest
+    needs: download-if-workflow-run
```

Comment:
Doesn't this mean `download-if-workflow-run` always has to run and complete?

Reply:
yes but as far as i can tell, in order to get an output from a previous step you must depend on that step

Reply:
so right now if its not a workflow run i skip all individual steps of `download-if-workflow-run` (but do not skip the job)